filter9000-boinc.cpp

```diff
@@ -289,15 +296,16 @@ int main(int argc, char **argv) {
             #endif
         }
     }
+
     #ifdef BOINC
         boinc_begin_critical_section();
     #endif
     time_t elapsed = (time(NULL) - start) + elapsed_chkpoint;
     double done = (double)total;
-    double speed = done / (double) elapsed;
-    fprintf(stderr, "\nSpeed: %.2lf/s\n", speed );
+    double speed = (done / (double) elapsed) * 65535;
+    fprintf(stderr, "\nSpeed: %.2lf worldseeds/s\n", speed );
     fprintf(stderr, "Done.\n");
-    fprintf(stderr, "Processed: %llu seeds in %.2lfs seconds.\n", total, (double) elapsed_chkpoint + (double) elapsed );
+    fprintf(stderr, "Processed: %llu input seeds in %.2lfs seconds.\n", total, (double) elapsed_chkpoint + (double) elapsed );
```

Comment:
If we were processing input seeds and showing stats for input seeds, why show world seeds in the previous print?

Confusing to show stats for 65536 times the work when we only tell them they processed a number 65536x smaller.